content/news/new-jury-advice.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Advice, Hints, and General Information from the NWERC Jury"
+date: 2022-11-05T09:00:00+01:00
+draft: false
+---
+The NWERC Jury have prepared a document containing advice and hints on understanding the problem format.
+It also contains general information on how submissions are judged during a contest,
+as well as tips about interactive problems.
```

Comment:
```suggestion
The NWERC Jury have prepared a hints document.
This contains advice on input and output formatting and how submissions are judged during a contest,
as well as tips about interactive problems.
```

Reply:
Verwoerd suggested to not refer to the document as "hints" in https://github.com/WISVCH/nwerc-2022/pull/34#discussion_r1009727750:

> They are more notes on how [to read] the problems then hints how to solve them.

I do agree that the structure of the news post can be improved, so then my suggestion becomes:

```suggestion
The NWERC Jury have prepared an advice document.
This contains information on input and output formatting and how submissions are judged during a contest,
as well as tips about interactive problems.
```

Reply:
Hrmm; I much prefer 'hints' since that's how it's always been called. 'advice document' sounds very weak :upside_down_face:. If anything, `tips` maybe. (but not `tips & tricks` :P)

Reply:
sg